pkg/pki/x509/sans.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package x509
+
+import (
+	"crypto/x509"
+
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
+)
+
+func GetSubjectAlternateNames(cert *x509.Certificate) []string {
```

Comment:
not blocking - out of curiousity is this needed in cosign or sigstore/sigstore?

Reply:
This came from Cosign, lemme add a TODO to use a shared version out of sigstore/sigstore

Reply:
oh whoops, forgot i did move it! wil use that